targets.py

```diff
@@ -27,3 +28,8 @@ def _volume_exists(self):
         ).stdout
         names = text.split('\n')
         return self.name in names
+
+    def before_restore(self):
+        if not self._volume_exists():
```

Comment:
What should we do if the volume already exists?  Overwrite, delete then overwrite, error?

Reply:
At the moment it'll just overwrite, which is the same thing it'll do with a directory. I've set the rsync `--delete` flag so it'll overwrite any changed files and delete any files that don't exist in the backup